plugins/digitalbitbox/digitalbitbox.py

```diff
@@ -39,6 +41,32 @@
 def to_hexstr(s):
     return binascii.hexlify(s).decode('ascii')
 
+def derive_keys(x):
+    h = Hash(x)
+    h = hashlib.sha512(h).digest()
+    return (h[:32],h[32:])
+
+def hmac_oneshot(key: bytes, msg: bytes, digest) -> bytes:
+    if hasattr(hmac, 'digest'):
+        # requires python 3.7+; faster
+        return hmac.digest(key, msg, digest)
+    else:
+        return hmac.new(key, msg, digest).digest()
+
+def EncodeAES_bytes(secret: bytes, msg: bytes) -> bytes:
+    assert_bytes(msg)
+    iv = bytes(os.urandom(16))
+    ct = aes_encrypt_with_iv(secret, iv, msg)
+    return iv + ct
+
+def DecodeAES_bytes(secret: bytes, ciphertext: bytes) -> bytes:
```

Comment:
Heh, looks good.  Makes me wonder if we should offer this more generic funciton and have our own `bitcoin.EncodeAES` & `bitcoin.DecodeAES` call the byte-level one...